CONTRIBUTING.md

```diff
@@ -104,13 +137,13 @@ The [Expect](https://jestjs.io/docs/expect) page is a good reference for the man
 
 #### Think of it as testing user-visible features
 
-- Tests that test low-level implementation details are hard to maintain over time. Instead, test user-visible features.
-- Try to think of the purpose of the code that has missing tests.
-  - For example, in `taskFromLine()` in `src/Commands/CreateOrEdit.ts` the comments are quite useful in terms of showing the different scenarios being considered. Something like:
+* Tests that test low-level implementation details are hard to maintain over time. Instead, test user-visible features.
+* Try to think of the purpose of the code that has missing tests.
+  * For example, in `taskFromLine()` in `src/Commands/CreateOrEdit.ts` the comments are quite useful in terms of showing the different scenarios being considered. Something like:
         _already a task line with a global filter, already a task line missing the global filter, already a task line and there is no global filter, already a bullet item, not in a bullet item_
-  - These then would be good tests to write: specifically, tests to check that each of those scenarios does actually behave as expected.
-  - And if the implementation changed in future, those tests would be extremely useful to the maintainer at the time.
-  - And if a new behaviour was added in future, it would be obvious how to add a new test for it.
+  * These then would be good tests to write: specifically, tests to check that each of those scenarios does actually behave as expected.
+  * And if the implementation changed in future, those tests would be extremely useful to the maintainer at the time.
+  * And if a new behaviour was added in future, it would be obvious how to add a new test for it.
```

Comment:
I though that this PR was probably ready to go, but now I see that something has changed all list lines starting `-` to `*` instead.

`*` is less convenient for editing in Obsidian, as starting a new line with a `*` puts in `**` for bold, and the second asterisk needs to be manually deleted.